client/checkout/platform-checkout/express-button/woopay-express-checkout-payment-method.js

```diff
@@ -34,7 +34,7 @@ const wooPayExpressCheckoutPaymentMethod = () => ( {
 			emailSelector="#email"
 		/>
 	),
-	canMakePayment: () => true,
+	canMakePayment: 'undefined' !== typeof wcpayConfig,
```

Comment:
```suggestion
    canMakePayment: () => 'undefined' !== typeof wcpayConfig,
```

apparently, `canMakePayment` should be a function, otherwise WooPay express button in Blocks checkout doesn't show up